index.js

```diff
@@ -190,12 +185,13 @@ function loadConfig(program) {
         saveMessages.push(`Exclusions: ${processed.diff} been saved to config.`)
       }
 
-      if (program.excludeRegex) {
-        parsedConfig.exclude_regex = program.excludeRegex
-        // strip leading and trailing / if exists
-        if (parsedConfig.exclude_regex.substring(0, 1) === '/' && parsedConfig.exclude_regex.substring(parsedConfig.exclude_regex.length - 1) === '/') {
-          parsedConfig.exclude_regex = parsedConfig.exclude_regex.substring(1, parsedConfig.exclude_regex.length - 1)
-        }
+      if (program.excludeRegex || !isNaN(process.env.LOGDNA_EXCLUDE_REGEX)) {
```

Comment:
what is the `isNaN` check here guarding against? also the testcase would fail this check if it's provided through an environment variable:

```js
!isNaN("/\bNOPE\b/") // => false
```

Reply:
yea, it seems like it's a bug, @darinspivey ?

Reply:
yeah, that's a bug.  My mind snapped I think.

Reply:
fixed